drivers/hmis_csv_twenty_twenty/app/models/hmis_csv_twenty_twenty/importer/enrollment.rb

```diff
@@ -56,6 +56,9 @@ def self.complex_validations
         {
           class: HmisCsvValidation::EntryAfterExit,
         },
+        {
+          class: HmisCsvValidation::UniqueHudKey,
+        },
       ]
```

Comment:
I like the idea, but it isn't setup to do that since it's not actually a child method, just included later so it gets run instead.  We could make a default_complex_validations that always runs, and then runs the sub-class ones.  Since there's only one override now, I'm tempted to leave it as is, at least until we convert to the 2022 CSV spec.